src/pudl/metadata/resources/mcoe.py

```diff
@@ -0,0 +1,224 @@
+"""Resource metadata for the MCOE (marginal cost of electricity) tables."""
+from typing import Any
+
+AGG_FREQS = ["yearly", "monthly"]
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = (
+    {
+        f"heat_rate_by_unit_{freq}": {
+            "description": (
+                f"{freq.title()} heat rate estimates by generation unit. Generation "
+                "units are identified by ``unit_id_pudl`` and are composed of a set of "
+                "interconnected boilers and generators."
```

Comment:
Need to investigate the unfiltered distribution of these heat rates and add caveats here explaining why some of them are insane.